learn/update_and_migration/updating.mdx

```diff
@@ -36,13 +36,124 @@ Once the project has been successfully updated, you will receive an email confir
 </Capsule>
 
-### Step 1: Export data
+### Dumpless upgrade <NoticeTag type="experimental" label="experimental" />
+
+Dumpless upgrades are available when upgrading from Meilisearch >=v1.12 to Meilisearch >=v1.13
+
+#### Step 1: Make a backup
+
+Dumpless upgrades are an experimental feature. Because of that, it may in rare occasions partially fail and result in a corrupted database. To prevent data loss, create a snapshot of your instance:
+
+```sh
+curl \
+  -X POST 'MEILISEARCH_URL/snapshots'
+```
+
+Once the snapshot is available in your [snapshots folder](/learn/self_hosted/configure_meilisearch_at_launch#snapshot-destination), proceed to the next step.
+
+### Step 2: Stop the Meilisearch instance
+
+Next, stop your Meilisearch instance.
+
+<Tabs.Container labels={['Local installation', 'Cloud platforms']}>
+
+<Tabs.Content label="Local installation">
+
+If you're running Meilisearch locally, stop the program by pressing `Ctrl + c`.
+
+</Tabs.Content>
+
+<Tabs.Content label="Cloud platforms">
+
+If you're running Meilisearch as a `systemctl` service, connect via SSH to your cloud instance and execute the following command to stop Meilisearch:
+
+```bash
+systemctl stop meilisearch
+```
+
+You may need to prefix the above command with `sudo` if you are not connected as root.
+
+</Tabs.Content>
+</Tabs.Container>
+
+#### Step 3: Install the new Meilisearch binary
+
+Install the latest version of Meilisearch using:
+
+<Tabs.Container labels={['Local installation', 'Cloud platforms']}>
+
+<Tabs.Content label="Local installation">
+
+```bash
+curl -L https://install.meilisearch.com | sh
+```
+
+</Tabs.Content>
+
+<Tabs.Content label="Cloud platforms">
+
+```sh
+# replace MEILISEARCH_VERSION with the version of your choice. Use the format: `vX.X.X`
+curl "https://github.com/meilisearch/meilisearch/releases/download/MEILISEARCH_VERSION/meilisearch-linux-amd64" --output meilisearch --location --show-error
+```
+
+</Tabs.Content>
+</Tabs.Container>
+
+Give execute permission to the Meilisearch binary:
+
+```
+chmod +x meilisearch
+```
+
+For **cloud platforms**, move the new Meilisearch binary to the `/usr/bin` directory:
+
+```
+mv meilisearch /usr/bin/meilisearch
+```
+
+#### Step 4: Relaunch Meilisearch
+
+Execute the command below to import the dump at launch:
+
+<Tabs.Container labels={['Local installation', 'Cloud platforms']}>
+
+<Tabs.Content label="Local installation">
+
+```bash
+./meilisearch --experimental-dumpless-upgrade 
+```
+
+</Tabs.Content>
+
+<Tabs.Content label="Cloud platforms">
+
+```sh
+meilisearch --experimental-dumpless-upgrade 
+```
+
+</Tabs.Content>
+</Tabs.Container>
+
+Meilisearch should launch normally and immediately create a new `UpgradeDatabase` task. This task is processed immediately and cannot be canceled. You may follow its progress by using the `GET /tasks` endpoint to obtain its `taskUid`, then querying `GET /tasks/TASK_UID`.
```

Comment:
```suggestion
Meilisearch should launch normally and immediately create a new `UpgradeDatabase` task. This task is processed immediately and cannot be canceled. You may follow its progress by using the `GET /tasks?types=UpgradeDatabase` endpoint to obtain its `taskUid`, then querying `GET /tasks/TASK_UID`.
```

This way even if there are a lot of enqueued tasks you'll see it.

We could also tell people to use the new batch route since this route will also give users the `progress`ion of the upgrade process:
```
GET /batches?types=UpgradeDatabase
```